jetty-core/jetty-session/pom.xml

```diff
@@ -69,5 +69,10 @@
       <artifactId>jetty-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
If you've moved this up to core then the dependency in the jetty-eeX-test-sessions-jdbc/pom.xml can be deleted.

Reply:
nope this will not work as `test` scope is not transitive